Cubical/Algebra/CommRing/Instances/Pointwise.agda

```diff
@@ -0,0 +1,32 @@
+{-# OPTIONS --safe #-}
+module Cubical.Algebra.CommRing.Instances.Pointwise where
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.HLevels
+
+open import Cubical.Algebra.CommRing.Base
+
+private
+  variable
+    ℓ : Level
+
+pointwiseRing : (X : Type ℓ) (R : CommRing ℓ) → CommRing ℓ
+pointwiseRing X R =
+  let open CommRingStr (snd R)
+      isSetX→R = isOfHLevelΠ 2 (λ _ → isSetCommRing R)
+  in (X → fst R) ,
+     (commringstr
```

Comment:
Probably better to use some copatterns here? Maybe at least for the outermost level (so not the `makeIsCommRing` part)